constants.py

```diff
@@ -3,8 +3,20 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
 DATA_DIR = os.path.join(ROOT_DIR, 'data')
 DATA_LIST_ALL = ['currency_price_tw', 'workdays']
+TRADING_INSTRUMENTS = {
+    ('currency', 'TW'):
+        ('USD', 'JPY', 'AUD', 'SEK', 'NZD', 'EUR', 'ZAR', 'GBP')
+}
+TRADING_CONFIGS = {
+    ('currency', 'TW'): {
+        'price_dataset': 'currency_price_tw',
+        'cost_object': thousandaire.trading_cost.CurrencyCost}
```

Comment:
Can we still call it `cost_function`? It is a function object, and its abstract meaning is function as well.
Especially in human readable configs, we should use abstract meanings instead of engineering meanings.